docs/guides/data-movement/copy-offload.md

```diff
@@ -16,10 +16,9 @@ Tools are available in the `nnf-dm` repository to assist in creating this certif
 kubectl get systemconfiguration
 ```
 
-From the `nnf-deploy` workarea, change into the `nnf-dm` directory and run `tools/mk-usercontainer-secrets.sh`:
+From the `nnf-deploy` workarea, run `tools/mk-usercontainer-secrets.sh`:
```

Comment:
Perhaps this doesn't matter for LLNL since they do not use `nnf-deploy install`, but I wonder if you should add or update that !IMPORT note below to make sure these secrets are available.

If you don't do that and the cert, then copy offload won't work. I think that should be made loud and clear.

Reply:
I've been working on a Proposal state check for this secret.

Would it make sense to just place a copy of nnf-dm/tools/mk-usercontainer-secrets.sh in the argocd-boilerplate/tools directory?